Auto-detect does not work for arm 32 builds

See https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1267, think auto-detect does not work as the value of java.vm.name is unexpected.

01:47:02  AUTO_DETECT is set to true
01:47:02  /home/jenkins/workspace/Grinder/openjdkbinary/j2sdk-image/bin/java -cp ./bin/TestKitGen.jar org.openj9.envInfo.EnvDetector JavaInfo
01:47:02  System.getProperty('os.name')=Linux
01:47:02  
01:47:02  System.getProperty('os.arch')=arm
01:47:02  
01:47:02  System.getProperty('java.fullversion')=null
01:47:02  
01:47:02  Cannot determine System.getProperty('os.arch')=arm
01:47:02  
01:47:02  System.getProperty('java.vm.name')=OpenJDK Zero VM
01:47:02  
01:47:02  run_configure.mk:29: recipe for target 'testconfig' failed
01:47:02  make: *** [testconfig] Error 1

--- a/src/org/openj9/envInfo/JavaInfo.java
+++ b/src/org/openj9/envInfo/JavaInfo.java
@@ -54,7 +54,7 @@ public class JavaInfo {
             spec += "_ppc";
         } else if (osArch.contains("s390x")) {
             spec += "_390";
-        } else if (osArch.contains("aarch") || (osArch.contains("arm")) {
+        } else if (osArch.contains("aarch") || osArch.contains("arm")) {
             spec += "_arm";
         } else if (osArch.contains("sparcv9")) {
             spec += "_sparcv9";

--- a/src/org/openj9/envInfo/JavaInfo.java
+++ b/src/org/openj9/envInfo/JavaInfo.java
@@ -54,7 +54,7 @@ public class JavaInfo {
             spec += "_ppc";
         } else if (osArch.contains("s390x")) {
             spec += "_390";
-        } else if (osArch.contains("aarch")) {
+        } else if (osArch.contains("aarch") || (osArch.contains("arm")) {
             spec += "_arm";
         } else if (osArch.contains("sparcv9")) {
             spec += "_sparcv9";

